kubedifflib/_images.py

```diff
@@ -27,7 +27,10 @@ def load_config(*paths):
             continue
         with open(path, 'r') as stream:
             data = yaml.safe_load(stream)
-        kube_obj = KubeObject.from_dict(data)
+        try:
+            kube_obj = KubeObject.from_dict(data)
+        except KeyError:
```

Comment:
Would it make sense to inform the user of issues during the process?

Reply:
I was thinking the same, honestly. Unfortunately we don't have logging or error control in the solution and it is something we need to improve. That's why I'm not sure if now is the right moment. 

Regarding the control of the dictionary, I really like it even though it doesn't show anything.

Most of the functionality now, continues when there is lack of data, which is this case:

```py
            if not data:
                continue
```

Reply:
A vote for that 👍🏻

Reply:
Super! Can you file an issue?

Reply:
Of course!